datahub-frontend/app/auth/sso/oidc/custom/CustomOidcClient.java

```diff
@@ -25,5 +31,8 @@ protected void clientInit() {
     defaultAuthenticator(new CustomOidcAuthenticator(getConfiguration(), this));
     defaultProfileCreator(new OidcProfileCreator<>(getConfiguration()));
     defaultLogoutActionBuilder(new OidcLogoutActionBuilder<>(getConfiguration()));
+    if (parseJwtAccessTokenEnabled) {
```

Comment:
Instead of doing this if here, I'd recommend always using the OidcAuthorizationGenerator but with the oidc configs passed in! 

The OidcAuthorizationGenerator can internally decide what to do based on configs

Reply:
Another benefit is you will not need to change the constructor for this class!

Reply:
Done